app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -477,6 +484,32 @@ def parse_stack_resource(resource, group)
         return uid, new_result
       end
 
+      def parse_managed_image(image)
+        uid = image.id.downcase
+
+        os = image.properties.storage_profile.try(:os_disk).try(:os_type) || 'unknown'
+
+        new_result = {
+          :type               => ManageIQ::Providers::Azure::CloudManager::Template.name,
+          :uid_ems            => uid,
+          :ems_ref            => uid,
+          :name               => image.name,
+          :description        => "#{image.resource_group}\\#{image.name}",
+          :location           => @ems.provider_region,
+          :vendor             => 'azure',
+          :raw_power_state    => 'never',
+          :template           => true,
+          :publicly_available => false,
+          :operating_system   => process_os(image),
+          :hardware           => {
+            :bitness  => 64,
+            :guest_os => OperatingSystem.normalize_os_name(os)
+          }
```

Comment:
@bronaghs I'm just aping the existing code. My assumption was that it creates DB relationships properly so that you can do `vm.operating_system` or `vm.hardware.guest_os`.